Exercise_1.java

```diff
@@ -1,3 +1,5 @@
+//Time Complexity = O(1)
+//Space Complexity = O(1)

```

Comment:
Please check the mentioned space complexity

Reply:
Is there any mistake in time complexity as per my knowledge I wrote, if there's any mistake suggest me with the correct answer time and space complexity for wrong answers

Reply:
Please try to mention the time and space complexity for each function instead of problem as a whole since this is an implementation problem. If you consider for single functions, then the mentioned TC and SC is correct. Here in your implementation, you are fixing the size of the stack to 1000 but in real world we can store more than 1000 items in a stack. So you can't say surely that SC is O(1).